### DISCUSSION TITLE
Standards/Specifications/Certifications To Model SCI After?

### DISCUSSION THREAD

**Discussion by will-iamalpine** [Green-Software-Foundation/sci#68] [General]
Are there any other 'gold standards' for certifications & standards we could borrow inspiration from? 

The [Green Grid](https://www.thegreengrid.org/en/get-involved/current-projects) involved the following metrics:  
- INFRASTRUCTURE UTILITY EFFECTIVENESS (IUE): how much design infrastructure capacity an operational data center is able to use. It will measure data centers for effectiveness with a range between 100% (indicating design capacity can be fully used) and 0% (not unusable) with a real IUE between 0% and 100% showing some level of infrastructure usage deficiency or stranded capacity. Data center owners/operators will be able to gauge capacity usage effectiveness and where inefficiencies are and how to find ways to improve resource efficiency.
- POWER METRICS FOR IT EQUIPMENT has ICT Capacity and Utilization Metrics 
-  [Open Standard for Data Center Availability (OSDA)](https://www.thegreengrid.org/osda-tool/dashboard.html) promises to modernize data center availability classification and rating, similar to how PUE modernized data center power usage

[ISO 14064](https://www.iso.org/fr/standard/66453.html) is the international standard and a reference when it comes to greenhouse gas (GHG) emission reporting. It was created to standardize practices and certify the results. It can build upon the accounting methodologies below.
[Bilan Carbone](https://www.associationbilancarbone.fr/wp-content/uploads/2018/03/bilan-carbone-v8-guide-methodologique-final.pdf) is the french accounting methodology, compliant with ISO 14064. This methodology was created by the French Agency for ecological transition (ADEME).

[Greenhouse Gas Protocol ](https://ghgprotocol.org/)was created by the World Resources Institute (WRI) and the World Business Council for Sustainable Development. The GHG Protocol works with NGOs and governments to build a credible and efficient GHG accounting methodology.

![image](https://user-images.githubusercontent.com/8934290/135958440-32df4131-a737-4d09-9c9f-8e1628234b73.png)
![image](https://user-images.githubusercontent.com/8934290/135958463-d27b03b0-68d0-4bfd-90de-d61a1e21663b.png)

**Comment by atg-abhishek** [comment]
This is a good discussion to get started @buchananwp - are you thinking in terms of adoption of those standards or in terms of how they've been written? It might be good if you can start to articulate here some of the characteristics that we'd want to have here to achieve that "gold standard". Thoughts?

**Reply by will-iamalpine** [reply]
I was thinking initially in terms of writing style guides we could borrow. 

I'm now thinking in our final standard that we should reference other standards to do the heavy lifting (e.g. ISO standard around embodied carbon discussed by @arexub). If we can point to a standard around each of the core pieces of the SCI, we should be in a good place.

**Reply by atg-abhishek** [reply]
Agreed on that @buchananwp - maybe in addition to pointing to standards, if we can also embed the characteristics directly in the SCI that will make it more wholesome?

**Reply by will-iamalpine** [reply]
Yes, i'd like to start with an agreed-upon list of golden standards, and then we can do a pass to embed those desired characteristics

**Reply by atg-abhishek** [reply]
Sounds good @buchananwp - let's list those out here then?
